db/sg_replicate_cfg.go

```diff
@@ -1123,11 +1125,21 @@ func (m *sgReplicateManager) UpsertReplication(ctx context.Context, replication
 		if validateErr != nil {
 			return true, validateErr
 		}
+		switch cluster.Replications[replication.ID].TargetState {
+		case ReplicationStateRunning:
+			// by default a replication will be created running, or an existing replication can go from stopped to running
+			auditEvents = append(auditEvents, base.AuditIDISGRStart)
+		case ReplicationStateStopped:
+			// if initial creation, then there is no starting
+		case ReplicationStateResetting:
+			// a replication can start in a resetting state, or go from stop->reset
+			auditEvents = append(auditEvents, base.AuditIDISGRReset)
+		}
```

Comment:
I'm not sure we actually want multiple events fired if someone upserts a config (i.e AuditIDISGRUpdate as well as AuditIDISGRStart).  I think it would be clearer to reserve AuditIDISGRStart/Stop/Reset for the PUT _replicationStatus endpoint only.

Reply:
Agreed, the payload can determine this.